What's New entry for threadsafety improvements (#11241)
astropy/astropy: docs/whatsnew/4.3.rst

Furthermore, by ensuring expensive calculations of the precession-nutation
matrix are only done once, the transformations have been sped up considerably.

.. _whatsnew-4.3-threadsafe:

> Improvements in making Astropy threadsafe
> =========================================

Reviewer: ```suggestion
Improvements in making Astropy thread-safe
==========================================
```

Author: done

> To help use of Astropy in software for the Square Kilometer Array, an effort

Reviewer 2: ```suggestion
To help use of Astropy in software for the Square Kilometre Array, an effort
```

Reviewer 2: I wouldn't say it's just for the SKA though. I work on the MeerKAT radio telescope where we're also going to want these fixes. Maybe something like "To help use of Astropy in multi-threaded environments (for example, with Dask), ..."

Author: Thanks, done

> has started to make Astropy threadsafe for operation that do not explicitly

Reviewer: ```suggestion
has been started to make parts of ``astropy`` thread-safe for operations that do not explicitly
```

Author: Mostly used - Used "has been started" but replaced "parts of"  with "the basic infrastructure of"

> change state. In particular, parsing of units and angles, and updating

Reviewer 2: ```suggestion
change state. In particular, parsing of units and angles, and initializing
```
Since it's the automatic initialization that's a problem, rather than any explicit update made by the user (not sure if you prefer US or UK spelling).

Reviewer: > prefer US or UK spelling

⚔️ Came up on Slack, I don't think we agreed on anything about this... ⚔️

Author: Being in Canada I never know which one to use...

> leap-second tables should now be threadsafe, as should any use of the

Reviewer: ```suggestion
leap-second tables should now be thread-safe, as should any use of the
```

Author: Thanks, rephrased slightly.
